README.md

```diff
@@ -36,7 +36,9 @@ The following common block must be filled in, for all usages of the projects.
 ```
 "KeyVault": {
     "KeyVaultUrl": "<Key Vault URL>",
-    "IssuingCA": "<Name of the issuing certificate in KeyVault.>"
+    "IssuingCA": "<Name of the issuing certificate in KeyVault.>",
+    "CertValidityInDays": "<Validity period for issued certificates (maximum is 365 days)>",
+    "CertPathLength": "<Path length of the certificate chain which gives the maximum number of non-self-issued intermediate certificates that may follow this certificate in a valid certification path. For the Root CA stored in KV, the default is 1, for the others is 0.>"
```

Comment:
So the current default value in the appsettings is 1, so it's correct for the Root CA stored in KV. Will user know that he/she should change the value to 0 when using the others?